src/graph_notebook/notebooks/04-Machine-Learning/Neptune-ML-06-Text-Encoding-Tutorial.ipynb

```diff
@@ -0,0 +1,1281 @@
+    "--job-id {training_job_name} \n",
+    "--s3-input-uri {export_results['outputS3Uri']} \n",
+    "--s3-processed-uri {str(s3_bucket_uri)}/preloading \"\"\"\n",
+    "print(processing_params)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 27,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "e66c3f30f0de43bfbdda1f7cc0ba6a52",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Output()"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "%neptune_ml dataprocessing start --wait --store-to processing_results {processing_params}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "{'id': 'link-prediction-1657824725', 'status': 'Completed', 'processingJob': {'name': 'link-pre-preloading-2022-07-14-18-52-6520000', 'arn': 'arn:aws:sagemaker:us-east-1:678790208956:processing-job/link-pre-preloading-2022-07-14-18-52-6520000', 'status': 'Completed', 'outputLocation': 's3://jenny-us-east-1/kaggle_job_recommend/preloading/link-pre-preloading-2022-07-14-18-52-6520000/preloading-output'}}\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(processing_results)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Model training\n",
+    "The model training is done in two stages. The first stage uses a SageMaker Processing job to generate a model training strategy, which is a configuration that specifies the type of model and the ranges to search for each of the training hyperparameters. This SageMaker Processing job uses the configuration to then launch a SageMaker Hyperparameter Optimization (HPO) job that will attempt to find the parameter values that maximize model performance. To do this, the SageMaker HPO job runs a pre-specified number of model training job trials to intelligently search the hyperparameter ranges for the optimal value.  The model artifacts of each training job are stored in the output S3 location and once all the training jobs are complete, the best performing model is chosen.\n",
+    "\n",
+    "Additional options and configuration parameters for training models can be found using the links below: [Model Training](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-on-graphs-model-training.html) and [modeltraining command](https://docs.aws.amazon.com/neptune/latest/userguide/machine-learning-api-modeltraining.html).\n",
+    "\n",
+    "<div style=\"background-color:#eeeeee; padding:10px; text-align:left; border-radius:10px; margin-top:10px; margin-bottom:10px; \"><b>Information</b>: Link prediction is a more computationally complex model than classification or regression so training this model will take 2-3 hours</div>"
```

Comment:
Confusing because at the beginning of this notebook, we say it will take 1 hour to complete this exercise. Please update one of the notes to be consistent with the other.

Reply:
I will remove this part. This comment is not correct here any more.